src/main/test/java/io/orangebeard/listener/OrangebeardTableLogParserTest.java

```diff
@@ -49,22 +44,19 @@ public void multiple_images_can_be_replaced() {
         try {
             File file = mock(File.class);
             PowerMockito.whenNew(File.class).withArguments(anyString()).thenReturn(file);
-            when(ImageEncoder.encode(any())).thenAnswer(new Answer<String>() {
-                @Override
-                public String answer(InvocationOnMock invocation) throws Throwable {
-                    Object[] args = invocation.getArguments();
-                    return "BASE64OF:" + args[0].toString();
-                }
+            when(ImageEncoder.encode(any())).thenAnswer((Answer<String>) invocation -> {
+                Object[] args = invocation.getArguments();
+                return "BASE64OF:" + args[0].toString();
             });
-        } catch (Exception e) {}
-        String result = parser.embedImagesAndStripHyperlinks(TEST_HTML_WITH_IMAGES);
+        } catch (Exception ignored) {}
```

Comment:
of de testmethode moet een exceptie gooien, of je moet met assertj de exceptie verwachten? 
```
assertThatThrownBy(() -> <methode>).isInstanceOf(SpecialException.class)
```